app/views/reports/regions/details.html.erb

```diff
@@ -49,14 +50,14 @@
                 Total registered<br>
                 patients
             </th>
-            <% dates_for_periods(@period.type, 6, include_current_period: @show_current_period).each do |date| %>
+            <% @period_range.each do |period| %>
               <th class="row-label sort-label sort-label-small" data-sort-method="number">
-                <%= format_period(@period.type, date) %>
+                <%= period %>
```

Comment:
Oof, nice. `to_s` kicks in as we'd like?

Reply:
yup, that's the default for ERB `<%= `